### PR TITLE
auto update interval

### PR DESCRIPTION
**Pull request by mikz**
set either `AUTO_UPDATE_INTERVAL` or `-i N` when starting APIcast and see it update every N seconds!

No idea how to test this tough.

### CODE REVIEW

File: apicast/conf/nginx.conf [hunk 1]
```diff
@@ -14,6 +14,8 @@ env REDIS_PORT;
 env RESOLVER;
 env APICAST_MODULE;
 env APICAST_MISSING_CONFIGURATION;
+env AUTO_UPDATE_INTERVAL;
+env PATH;
```
Comment (davidor) [inline apicast/conf/nginx.conf]: Where is this used? It's not related to the auto update feature, is it?

File: apicast/src/apicast.lua [hunk 2]
```diff
@@ -30,6 +31,41 @@ function _M.init()
   end
 end
 
+function _M.init_worker()
+  local interval = tonumber(os.getenv('AUTO_UPDATE_INTERVAL'), 10)
```
Comment (davidor) [inline apicast/src/apicast.lua]: I'd extract that 10 to a constant: `DEFAULT_UPDATE_INTERVAL`. Also, we are using 10 as the default interval here and 60 in the `.env` file. I'd rather see the same value in both places for consistency.

**Comment by mikz** [comment]
This is not a default value, but integer base.

File: apicast/src/apicast.lua (hunk 2 above)
Comment (mikz) [inline apicast/src/apicast.lua]: This is not a default value, but integer base.

Comment (davidor) [inline apicast/src/apicast.lua]: Right.
Do we need to specify that 10? It's the default value and what everyone would expect. Am I missing something?

Comment (mikz) [inline apicast/src/apicast.lua]: For example in javascript it does not work like that. So honestly I don't know what to expect :)

**Pull request merged**